runtime/src/main/java/com/dylibso/chicory/runtime/HostGlobal.java

```diff
@@ -20,12 +25,34 @@ public HostGlobal(String moduleName, String fieldName, Value value, MutabilityTy
         this.fieldName = fieldName;
     }
 
+    public HostGlobal(
```

Comment:
We are defining a "proxy" `HostGlobal`, this is one possible way to implement the functionality.